pandas/core/resample.py

```diff
@@ -418,23 +418,80 @@ def pad(self, limit=None):
 
     def nearest(self, limit=None):
         """
-        Fill values with nearest neighbor starting from center
+        Resample by using the nearest value.
+
+        When resampling data, missing values may appear (e.g., when the
+        resampling frequency is higher than the original frequency).
+        The nearest fill will replace ``NaN`` values that appeared in
```

Comment:
"nearest fill" refers to this method, right?  Maybe make that explicit with "`nearest` fill method"?